links/gql_websocket_link/lib/src/messages.dart

```diff
@@ -165,7 +165,7 @@ class SubscriptionData extends GraphQLSocketMessage {
   int get hashCode => toJson().hashCode;
 
   @override
-  bool operator ==(dynamic other) =>
+  bool operator ==(Object other) =>
```

Comment:
Is this an unrelated change?

Reply:
Yes. To be honest I cannot figure out why but the analyzer was failing 
<img width="1418" alt="image" src="https://github.com/gql-dart/gql/assets/30510627/be8a8539-35d1-41af-bb41-f267d1858bc7">

Your commit was the last one and then it was succeeding. Now it doesn't 🤷‍♂️

Reply:
Maybe because workflow uses `dart:latest`, so something in the Dart could change.
<img width="414" alt="image" src="https://github.com/gql-dart/gql/assets/30510627/76a47dc7-9359-4c86-82af-c8f0f6279d32">

Reply:
yes, it's likely a new lint rule in dart

Reply:
I couldn't find that 😅